dask/blockwise.py

```diff
@@ -354,7 +354,8 @@ def __dask_distributed_unpack__(cls, state, dsk, dependencies, annotations):
         global_dependencies = list(state["global_dependencies"])
 
         if state["annotations"]:
-            annotations.update(cls.expand_annotations(state["annotations"], raw.keys()))
+            expanded = cls.expand_annotations(state["annotations"], raw.keys())
+            cls.merge_annotations(annotations, expanded)
```

Comment:
It's not a blocker for merging but with the changes here and in https://github.com/dask/distributed/pull/4406 it looks like we never call `expand_annotations` without a subsequent call to `merge_annotations`. Should we combine them into a single method to avoid accidentally calling one without the other?

Reply:
Yeah, my thought had been to avoid the breaking API change, but perhaps the danger of splitting the two functions up is the greater concern.

Reply:
I agree especially as this will be less of a problem once the API changes mentioned here are implemented: https://github.com/dask/distributed/pull/4406#discussion_r563618160. Annotations would be expanded (and merged) in Layer.__dask_unpack_annotations__ and derived classes would defer to this implementation in the majority of cases.

Reply:
Okay, I've combined them into a single method